### PR TITLE
 Update active version information for 2020

### PR DESCRIPTION
**Pull request by kyordhel**
Changes:
- Changed version to Draft
- Changed year to 2020
- Updated citation information
- Updated acknowledgements
- Updated infrastructure information
    - Update league's website
    - Add Telegram group
    - Add Rulebook repository
    - Update wiki page
- Updated committee members

### CODE REVIEW

File: citation.bib [hunk 1]
```diff
@@ -1,10 +1,8 @@
-@misc{rulebook_2019,
-  author =       {Matamoros, Mauricio and Rascon, Caleb and Wachsmuth, Sven and
-                  Moriarty, Alexander William and Kummert, Johannes and
-                  Hart, Justin and Pfeiffer, Sammy and
-                  {van der Brugh}, Matthijs and St-Pierre, Maxime},
-  title =        {RoboCup@Home 2019: Rules and Regulations (draft)},
-  year =         {2019},
-  howpublished = {\url{http://www.robocupathome.org/rules/2019_rulebook.pdf}},
+@misc{rulebook_2020,
+  author =       {Matamoros, Mauricio and Moriarty, Alexander William and
```
Comment (moriarty) [inline citation.bib]:
```suggestion
  author =       {Matamoros, Mauricio and Moriarty, Alexander Moriarty and
```

File: introduction/Infrastructure.tex [hunk 2]
```diff
@@ -9,23 +9,42 @@ \section{Infrastructure}
...
 \subsection{RoboCup@Home Web Page}
-The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at \\
+The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at:
 \begin{center}
-http://www.robocupathome.org/
+{\small\url{https://athome.robocup.org/}}
+\end{center}
+
+\subsection{RoboCup@Home Rulebook Repository}
+The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes, buffs, and nerfs to the rulebook.
```
Comment (moriarty) [inline introduction/Infrastructure.tex]:
```suggestion
The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes to the rulebook.
```

Buffs is informal, and nerfs isn't a word. Let's just leave it at any changes.

File: introduction/Infrastructure.tex [hunk 3]
```diff
@@ -9,23 +9,42 @@ \section{Infrastructure}
...
+The entire RoboCup@Home community is welcome and encouraged to actively participate in rule discussion and crafting. The repository can be reached at:
```
Comment (moriarty) [inline introduction/Infrastructure.tex]:
```suggestion
The entire RoboCup@Home community is welcome and encouraged to actively participate in creating and discussing the rules. The repository can be reached at:
```

I don't like the word crafting. I used **google** `define: create` and `define: crafting`

```
cre·ate
/krēˈāt/
verb
gerund or present participle: creating
bring (something) into existence.
```

```
craft·ing
/ˈkraftiNG/
noun
the activity or hobby of making decorative articles by hand.
```

Comment (kyordhel) [reply introduction/Infrastructure.tex]: Time to go back to school and learn English again...

**Comment by kyordhel** [comment]
Solved!

File: introduction/Infrastructure.tex [hunk 4]
```diff
@@ -9,23 +9,42 @@ \section{Infrastructure}
...
+The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes to the rulebook.
+The entire RoboCup@Home community is welcome and encouraged to actively participate in creating and discussing the rules. The repository can be reached at:
+\begin{center}
+{\small\url{https://github.com/RoboCupAtHome/RuleBook/}}
+\end{center}
+
+Although opening issues with inconsistencies, questions, clarifications, and suggestions is highly appreciated, the best way to contribute is by making pull requests with fixes and proposed changes.
+
+\subsection{RoboCup@Home Telegram Group}
+The official \iterm{RoboCup@Home Telegram Group} is and communication channel for the RoboCup@Home community where rules are discussed, announcements are made, and questions are answered.
```
Comment (johaq) [inline introduction/Infrastructure.tex]: Small typo:  The official \iterm{RoboCup@Home Telegram Group} is **a** communication

File: introduction/Infrastructure.tex [hunk 5]
```diff
@@ -9,23 +9,42 @@ \section{Infrastructure}
...
+However, beyond the technical aspects of the competition, the \textit{Telegram Group} is a meeting point to stay in contact with the community, foster knowledge exchange and strengthen relationships.
```
Comment (johaq) [inline introduction/Infrastructure.tex]: Maybe: In addition to discussing technical aspects of the competition, ...

However sounds like this is opposed to what was stated before.

File: introduction/Organization.tex [hunk 6]
```diff
@@ -35,18 +38,21 @@ \subsection{Organizing Committee --- oc@robocupathome.org}
...
-	\item Jeffrey Too Chuan Tan (Nankai University, China)
-	\item Jesus Savage (University of Mexico, Mexico)
-	\item João Cartucho (University of Lisbon, Portugal)
-	\item Luz Martínez (Universidad de Chile, Chile)
-	\item Nathalie Connolly (Concordia University, Canada)
-	\item Raphael Memmesheimer (University of Koblenz, Germany)
+	\item Colm Flanagan (University New South Wales, Australia)
+	\item David Garcia (Colombia)
+	\item Fagner Pimentel (FEI University Center, Brazil)
+	\item Florian Lier (University of Bielefeld, Germany)
```
Comment (johaq) [inline introduction/Organization.tex]: Florian Lier (Bielefeld University, Germany)

File: introduction/Organization.tex [hunk 7]
```diff
@@ -35,18 +38,21 @@ \subsection{Organizing Committee --- oc@robocupathome.org}
...
+	\item Jeffrey Cousineau (Canada)
+	\item Jose Garcia (Colombia)
+	\item Shengxin Luo (University of Miami, USA)
+	\item Sven Wachsmuth (University of Bielefeld, Germany)
```
Comment (johaq) [inline introduction/Organization.tex]: Sven Wachsmuth (Bielefeld University, Germany)

File: introduction/Infrastructure.tex (hunk 4 above)
Comment (moriarty) [reply introduction/Infrastructure.tex]:
```suggestion
The official \iterm{RoboCup@Home Telegram Group} is a communication channel for the RoboCup@Home community where rules are discussed, announcements are made, and questions are answered.
```

File: introduction/Organization.tex (hunk 6 above)
Comment (moriarty) [reply introduction/Organization.tex]:
```suggestion
	\item Florian Lier (Bielefeld University, Germany)
```

File: introduction/Organization.tex (hunk 7 above)
Comment (moriarty) [reply introduction/Organization.tex]:
```suggestion
	\item Sven Wachsmuth (Bielefeld University, Germany)
```

File: introduction/Infrastructure.tex (hunk 5 above)
Comment (kyordhel) [reply introduction/Infrastructure.tex]: Any suggestions?

File: introduction/Organization.tex (hunk 6 above)
Comment (kyordhel) [reply introduction/Organization.tex]: Really? It has been like that for years!

Comment (moriarty) [reply introduction/Organization.tex]: Haha yeh I thought the same. But @johaq went there so I assume he knows?

Comment (moriarty) [reply introduction/Organization.tex]: https://en.m.wikipedia.org/wiki/Bielefeld_University

Comment (moriarty) [reply introduction/Organization.tex]: No you don’t always add the “of” it’s either part of the university’s name or it isn’t.

Looks like we’ve always had it wrong.

Comment (moriarty) [reply introduction/Organization.tex]: No you don’t always add the “of” it’s either part of the university’s name or it isn’t.

Looks like we’ve always had it wrong.

File: introduction/Infrastructure.tex (hunk 5 above)
Comment (nickswalker) [reply introduction/Infrastructure.tex]:
```suggestion
Beyond the technical aspects of the competition, the \textit{Telegram Group} is a meeting point to stay in contact with the community, foster knowledge exchange and strengthen relationships.
```

**Comment by kyordhel** [comment]
All requested changes done.

Would have pushed them earlier, but got 500

**Pull request merged**